### ISSUE TITLE
FoxPro9 Memo fields are 4 bytes in size

### ISSUE THREAD

**Issue by Khaos66** [yortus/DBFFile#57]
When trying to open a vfp9 DBF, which contains at least one field of type Memo, then this lib throws this error: 
`Invalid field size (must be 10)`

The size 10 is hard-coded inside field-descriptor.ts
```ts
if (type === 'M' && size !== 10) throw new Error('Invalid field size (must be 10)');
```

I'm not sure if this is different from other dBase DBFs, but as far as I know vfp9 always uses the size of 4 bytes for Memo fields

If this is different, maybe there could be a `dialect` option?

**Comment by Khaos66** [comment]
Oh dear... #19 looks like some ppl had the same issue but failed to solve it some time ago... I guess I'm on my own now?

**Comment by Khaos66** [comment]
@yortus please see PR #58 for this issue. Can you merge and release it please?

**Comment by iwkse** [comment]
Any chance to merge PR #58?
Thank you

**Comment by yortus** [comment]
Sorry for the delay - I had GH notifications turned off for a couple of months without realising. PR #58 has been merged and new version 1.8.0 has been published to npm.
